### PR TITLE
Transformations : Add partition by values to transformation docs

### PR DESCRIPTION
**Pull request by bohandley**
**What is this feature?**
Adds description of new transformation, [Partition by Values](https://github.com/grafana/grafana/pull/56767), to the docs. 

This is the same info written by @leeoniya  https://docs.google.com/document/d/14n15xfUgiTqTHKfnePJnMU16MSw0xYncXTOR1KV9MpE/edit# formatted into markdown.

Thank you!

### CODE REVIEW

File: docs/sources/panels-visualizations/query-transform-data/transform-data/index.md [hunk 1]
```diff
@@ -501,6 +501,40 @@ In the example below, I hid the value field and renamed Max and Min.
...
+This transformation can help eliminate the need for multiple queries to the same datasource with different WHERE clauses when graphing multiple series. Consider a metrics SQL table with the following data:
+
+| Time                | Region | Value |
+| ------------------- | ------ | ----- |
+| 2022-10-20 12:00:00 | US     | 1520  |
+| 2022-10-20 12:00:00 | EU     | 2936  |
+| 2022-10-20 01:00:00 | US     | 1327  |
+| 2022-10-20 01:00:00 | EU     | 912   |
+
+Prior to v9.3. if you wanted to plot a red trendline for US and a blue one for EU in the same TimeSeries panel, you would likely have to split this into two queries:
```
Comment (knylander-grafana) [inline docs/sources/panels-visualizations/query-transform-data/transform-data/index.md:517]: Changing a period to a comma. 
```suggestion
Prior to v9.3, if you wanted to plot a red trendline for US and a blue one for EU in the same TimeSeries panel, you would likely have to split this into two queries:
```

File: docs/sources/panels-visualizations/query-transform-data/transform-data/index.md [hunk 2]
```diff
@@ -501,6 +501,40 @@ In the example below, I hid the value field and renamed Max and Min.
 
 {{< figure src="/static/img/docs/transformations/organize-fields-stat-example-7-0.png" class="docs-image--no-shadow" max-width= "1100px" >}}
 
+### Partition by values
+
+Available in experimental in Grafana Open Source, Enterprise, Cloud Pro, Cloud Advanced
```
Comment (knylander-grafana) [inline docs/sources/panels-visualizations/query-transform-data/transform-data/index.md:506]:
```suggestion
Available in experimental in Grafana Open Source, Enterprise, Cloud Pro, and Cloud Advanced.
```

File: docs/sources/panels-visualizations/query-transform-data/transform-data/index.md [hunk 3]
```diff
@@ -501,6 +501,40 @@ In the example below, I hid the value field and renamed Max and Min.
 
 {{< figure src="/static/img/docs/transformations/organize-fields-stat-example-7-0.png" class="docs-image--no-shadow" max-width= "1100px" >}}
 
+### Partition by values
+
+Available in experimental in Grafana Open Source, Enterprise, Cloud Pro, and Cloud Advanced.
+
+This transformation can help eliminate the need for multiple queries to the same datasource with different WHERE clauses when graphing multiple series. Consider a metrics SQL table with the following data:
```
Comment (knylander-grafana) [inline docs/sources/panels-visualizations/query-transform-data/transform-data/index.md:508]:
```suggestion
This transformation can help eliminate the need for multiple queries to the same datasource with different `WHERE` clauses when graphing multiple series. Consider a metrics SQL table with the following data:
```

File: docs/sources/panels-visualizations/query-transform-data/transform-data/index.md [hunk 4]
```diff
@@ -501,6 +501,40 @@ In the example below, I hid the value field and renamed Max and Min.
...
+| 2022-10-20 01:00:00 | EU     | 912   |
+
+Prior to v9.3, if you wanted to plot a red trendline for US and a blue one for EU in the same TimeSeries panel, you would likely have to split this into two queries:
+
+`SELECT Time, Value FROM metrics WHERE Time > ‘2022-10-20’ AND Region=’US’`
+`SELECT Time, Value FROM metrics WHERE Time > ‘2022-10-20’ AND Region=’EU’`
+
+This also requires you to know ahead of time which regions actually exist in the metrics table.
+
+With the Partition by values transformer, you can now issue a single query and split the results by unique (enum) values from one or more columns (fields) of your choosing. In this case, Region.
```
Comment (knylander-grafana) [inline docs/sources/panels-visualizations/query-transform-data/transform-data/index.md:524]:
```suggestion
With the Partition by values transformer, you can now issue a single query and split the results by unique (`enum`) values from one or more columns (`fields`) of your choosing. The following example uses `Region`.
```

**Review by knylander-grafana — APPROVED** [review]
I added a few basic edits. Looks like good doc!

**Comment by bohandley** [comment]
Thank you @knylander-grafana! I appreciate the feedback and the quick review

File: docs/sources/panels-visualizations/query-transform-data/transform-data/index.md [hunk 5]
```diff
@@ -501,6 +501,40 @@ In the example below, I hid the value field and renamed Max and Min.
 
 {{< figure src="/static/img/docs/transformations/organize-fields-stat-example-7-0.png" class="docs-image--no-shadow" max-width= "1100px" >}}
 
+### Partition by values
+
+Available in experimental in Grafana Open Source, Enterprise, Cloud Pro, and Cloud Advanced.
```
Comment (leeoniya) [inline docs/sources/panels-visualizations/query-transform-data/transform-data/index.md:506]: this wording is bizzare. can't we just say "Everywhere"?

File: docs/sources/panels-visualizations/query-transform-data/transform-data/index.md [hunk 6]
```diff
@@ -501,6 +501,40 @@ In the example below, I hid the value field and renamed Max and Min.
...
+| Time                | Region | Value |
+| ------------------- | ------ | ----- |
+| 2022-10-20 12:00:00 | US     | 1520  |
+| 2022-10-20 12:00:00 | EU     | 2936  |
+| 2022-10-20 01:00:00 | US     | 1327  |
+| 2022-10-20 01:00:00 | EU     | 912   |
+
+Prior to v9.3, if you wanted to plot a red trendline for US and a blue one for EU in the same TimeSeries panel, you would likely have to split this into two queries:
+
+`SELECT Time, Value FROM metrics WHERE Time > ‘2022-10-20’ AND Region=’US’`
```
Comment (leeoniya) [inline docs/sources/panels-visualizations/query-transform-data/transform-data/index.md:519]: i think this needs a `<br>` after it.

File: docs/sources/panels-visualizations/query-transform-data/transform-data/index.md [hunk 7]
```diff
@@ -501,6 +501,39 @@ In the example below, I hid the value field and renamed Max and Min.
...
+
+`SELECT Time, Value FROM metrics WHERE Time > ‘2022-10-20’ AND Region=’EU’`
```
Comment (leeoniya) [inline docs/sources/panels-visualizations/query-transform-data/transform-data/index.md:519]:
```suggestion
`SELECT Time, Value FROM metrics WHERE Time > '2022-10-20' AND Region='US'`<br>
`SELECT Time, Value FROM metrics WHERE Time > '2022-10-20' AND Region='EU'`
```

File: docs/sources/panels-visualizations/query-transform-data/transform-data/index.md [hunk 8]
```diff
@@ -501,6 +501,38 @@ In the example below, I hid the value field and renamed Max and Min.
...
+Prior to v9.3, if you wanted to plot a red trendline for US and a blue one for EU in the same TimeSeries panel, you would likely have to split this into two queries:
+
+`SELECT Time, Value FROM metrics WHERE Time > '2022-10-20' AND Region='US'`<br>
+`SELECT Time, Value FROM metrics WHERE Time > '2022-10-20' AND Region='EU'`
+
+This also requires you to know ahead of time which regions actually exist in the metrics table.
+
+With the _Partition by values_ transformer, you can now issue a single query and split the results by unique values (often of the enum type) from one or more columns (`fields`) of your choosing. The following example uses `Region`.
+
+`SELECT Time, Region, Value FROM metrics WHERE Time > ‘2022-10-20’`
```
Comment (leeoniya) [inline docs/sources/panels-visualizations/query-transform-data/transform-data/index.md:524]:
```suggestion
`SELECT Time, Region, Value FROM metrics WHERE Time > '2022-10-20'`
```

**Comment by leeoniya** [comment]
LGTM if we're fine using "you" in the docs (not sure if this matches the style of the existing docs)

**Review by leeoniya — APPROVED** [review]

File: docs/sources/panels-visualizations/query-transform-data/transform-data/index.md [hunk 9]
```diff
@@ -501,6 +501,38 @@ In the example below, I hid the value field and renamed Max and Min.
...
+| 2022-10-20 01:00:00 | EU     | 912   |
+
+Prior to v9.3, if you wanted to plot a red trendline for US and a blue one for EU in the same TimeSeries panel, you would likely have to split this into two queries:
+
+`SELECT Time, Value FROM metrics WHERE Time > '2022-10-20' AND Region='US'`<br>
+`SELECT Time, Value FROM metrics WHERE Time > '2022-10-20' AND Region='EU'`
+
+This also requires you to know ahead of time which regions actually exist in the metrics table.
+
+With the _Partition by values_ transformer, you can now issue a single query and split the results by unique values (often of the enum type) from one or more columns (`fields`) of your choosing. The following example uses `Region`.
```
Comment (leeoniya) [inline docs/sources/panels-visualizations/query-transform-data/transform-data/index.md:522]:
```suggestion
With the _Partition by values_ transformer, you can now issue a single query and split the results by unique values in one or more columns (`fields`) of your choosing. The following example uses `Region`.
```

**Comment by bohandley** [comment]
> LGTM if we're fine using "you" in the docs (not sure if this matches the style of the existing docs)

Stylistically, "you" is all over the place in this file, even in the intro.

"Transformations are a powerful way to manipulate data returned by a query before the system applies a visualization. Using transformations, _you_ can:"

**Pull request merged**